modules/channel-vonage/src/backend/client.ts

```diff
@@ -296,9 +298,21 @@ export class VonageClient {
 
       if (picture) {
         await this.sendImage(event, { url: picture, type: 'image' })
+
+        // Sandbox API is limited to one call per second
+        if (this.config.useTestingApi === true) {
+          await Promise.delay(1000)
+        }
       }
 
       await this.sendOptions(event, body, options)
+
+      // Sandbox API is limited to one call per second. Wait one second between calls
+      if (this.config.useTestingApi === true && counter < payload.elements.length - 1) {
+        await Promise.delay(1000)
+      }
```

Comment:
can you hide this logic as well in the send options function

Reply:
Same answer as above!